xml/System.Web.Configuration/BrowserCapabilitiesFactory.xml

```diff
@@ -67,12 +68,13 @@
         <Parameter Name="browserCaps" Type="System.Web.HttpBrowserCapabilities" />
       </Parameters>
       <Docs>
-        <param name="ignoreApplicationBrowsers">To be added.</param>
-        <param name="headers">To be added.</param>
-        <param name="browserCaps">To be added.</param>
-        <summary>To be added.</summary>
+        <param name="ignoreApplicationBrowsers"><see langword="true" /> to ignore definitions for the A500 browser in application-level browser definition files; otherwise, <see langword="false" />.</param>
+        <param name="headers">The collection of headers included in the current request.</param>
+        <param name="browserCaps">The <see cref="T:system.web.httpbrowsercapabilities"/> object to initialize.</param>
+        <summary>This API supports the .NET Framework infrastructure and is not intended to be used directly from your code. Initializes an <see cref="T:system.web.httpbrowsercapabilities"/> object that represents the capabilities of the A500 browser making the current request.</summary>
```

Comment:
Please remove "This API supports the .NET Framework infrastructure and is not intended to be used directly from your code. " from the summaries. Also, your links are still broken. note the capitalization, it's case sensitive.
system.web.httpbrowsercapabilities -> System.Web.HttpBrowserCapabilities

```suggestion
        <summary>Initializes an <see cref="T:System.Web.HttpBrowserCapabilities"/> object that represents the capabilities of the A500 browser making the current request.</summary>
```